backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/descriptors/DescriptorUtils.kt

```diff
@@ -74,9 +69,14 @@ internal fun <T : CallableMemberDescriptor> T.resolveFakeOverride(): T {
 
 private val intrinsicAnnotation = FqName("konan.internal.Intrinsic")
 
-// TODO: check it is external?
 internal val CallableDescriptor.isIntrinsic: Boolean
-    get() = this.annotations.findAnnotation(intrinsicAnnotation) != null
+    get() = when {
```

Comment:
Unneeded.

Reply:
At first I added the check for isExternal for FunctionDescriptor, but one of Igor's commits changed this extension to CallableDescriptor, which is not applicable for the isExternal check.
I'll remove this change